apps/examples/testcase/ta_tc/systemio/itc/Kconfig

```diff
@@ -14,20 +14,79 @@ config SYSIO_ITC_UART
 	bool "Sysio UART TC"
 	default n
 
+if SYSIO_ITC_UART
+config SYSIO_ITC_UART_FILE_PATH
+	string "Sysio UART virtual test port"
+	default "/dev/ttyS1"
+endif
+
 config SYSIO_ITC_SPI
 	bool "Sysio SPI TC"
 	default n
 
+if SYSIO_ITC_SPI
+config SYSIO_ITC_SPI_BUS
+	int "Sysio SPI test bus"
+	default 0
+
+config SYSIO_ITC_SPI_BITS_PER_WORD
+	int "Sysio SPI test bits per word"
+	default 8
+
+config SYSIO_ITC_SPI_CHIP_SELECT
+	int "Sysio SPI test chip select"
+	default 0
+
+config SYSIO_ITC_SPI_FREQUENCY
+	int "Sysio SPI test frequency"
+	default 12000000
+
+config SYSIO_ITC_SPI_MODE
+	int "Sysio SPI test mode"
+	default 0
```

Comment:
The spi config in the driver has those configs. Do we need to set them separately?

Reply:
Do you mean 
spi_mode_e > SPIDEV_MODE0 ??
I can only see this in "include/tinyara/spi/spi.h"
Other values are not set in this